src/index.jsx

```diff
@@ -4,6 +4,17 @@ import {BrowserRouter} from 'react-router-dom'
 import BaseRoutes from './BaseRoutes'
 
 
+if (process.env.NODE_ENV === 'development' || process.env.NODE_ENV === 'test') {
```

Comment:
Per convo, can this be injected only in dev from something clean in esbuild?  Would be nice to not have this present in prod.

Reply:
I've confirmed that this is already optimized out by esbuild's tree shaking algorithm.

At build time, the value of `process.env.NODE_ENV` in any bundled JavaScript is evaluated to a value of `development` or `production` depending on whether minification has been requested.

https://github.com/evanw/esbuild/blob/master/pkg/api/api_impl.go#L552..L572

In #367, I introduced conditional minification depending on the value of the `NODE_ENV` environment variable because I wanted to be better troubleshoot issues in non-production environments.

https://github.com/bldrs-ai/Share/blob/main/config/common.js#L19

Then because the values are known are build time, the expression is evaluated down to `true` or `false`. If false, esbuild throws it out.

Here's the evidence in the form of the code result in a non-production environment (from the build artifact: `docs/index.js`):

![image](https://user-images.githubusercontent.com/113154043/195380858-1a4d1594-5ad4-4a0a-be61-ded5493b4a49.png)